README.md

```diff
@@ -678,7 +678,8 @@ The encrypted binary will have the following structure:
 - Padding to ensure the encrypted length is a multiple of 4 words
 - Signature metadata block
 
-The AES key must be provided as a .bin file of the 256 bit AES key to be used for encryption.
+The AES key must be provided as a .bin file containing a 4-way share of the 256 bit AES key to be used for encryption.
```

Comment:
Is there an (e.g.) `openssl` command that can be used to create this "4-way share of the 256 bit AES key" ?

Reply:
It’s just a 1024 bit random number, rather than a 256 bit one - see https://github.com/raspberrypi/pico-examples/pull/619 for generation suggestions

Reply:
Ahhh, I'd misinterpreted this to mean that the 1024-bit keyshare was directly generated from the 256-bit key; rather than the 256-bit key being generated from the (random) 1024-bit keyshare :upside_down_face:  :joy: